airflow/providers/google/suite/hooks/calendar.py

```diff
@@ -0,0 +1,213 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains a Google Calendar API hook"""
+
+from datetime import datetime
+from typing import Any, Dict, Optional, Sequence, Union
+
+from googleapiclient.discovery import build
```

Comment:
Do we want to use discovery?
see also https://github.com/apache/airflow/pull/20377#discussion_r774162313

Reply:
@eladkal is there dedicated calendar SDK? Google docs provide only example for discovery and no mention of dedicated client:
https://developers.google.com/calendar/api/quickstart/python

Reply:
Actually I didnt check :)
@rsg17 if there isnt llease ignore this comment :)